packages/xod-project/src/index.js

```diff
@@ -48,6 +48,11 @@ export {
   applyNodeIdMap,
   resolveNodeTypesInPatch,
   listLibraryNamesUsedInPatch,
+  computeVariadicPins,
+  listValueInputPins,
```

Comment:
Think of the name “list value input pins” in a vacuum? Isn’t it too generic? Maybe the better names are:

```
listVariadicInputPins,
listVariadicAccPins,
listVariadicSharedPins,
```